poc/prg.py

```diff
@@ -104,9 +104,12 @@ class PrgSha3(Prg):
     test_vec_name = 'PrgSha3'
 
     def __init__(self, seed, dst, binder):
-        # `dst` is used as the customization string; `seed || binder` is
+        # `dst` is used for domain separation; `seed || binder` is
         # used as the main input string.
```

Comment:
Since there's no separate customization string, we don't have a distinction to draw here anymore.
```suggestion
        # The input is composed of `dst`, the domain separation tag, the
        # `seed`, and the `binder` string.
```